nodes/modifier_make/cut_object_by_surface.py

```diff
@@ -249,7 +291,10 @@ def remember_connection(edges_by_face, bm_edge, bm_intersection):
 
                 edges_to_split = []
                 for new_vert_1, new_vert_2 in new_verts_by_old_face.values():
-                    edge = bm_obj.edges.get((new_vert_1, new_vert_2))
+                    try:
+                        edge = bm_obj.edges.get((new_vert_1, new_vert_2))
+                    except ValueError:
+                        continue
                     edges_to_split.append(edge)
```

Comment:
IMHO it would be more readable if you move this line inside the try block.

Reply:
Corrected in new commit.